### PR TITLE
set clear attribute

### PR DESCRIPTION
**Pull request by SadriG91**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@SadriG91
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/gh-oidc/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ module "gh_oidc" {
   sa_mapping = {
     "foo-service-account" = {
       sa_name   = "projects/my-project/serviceAccounts/foo-service-account@my-project.iam.gserviceaccount.com"
-      attribute = "attribute.repository/user/repo"
+      attribute = "attribute.repository/<user/org>/<repo>"
```
Comment (bharathkkb) [inline modules/gh-oidc/README.md:22]: Perhaps adding it in upper case may make it clearer.
```suggestion
      attribute = "attribute.repository/${USER/ORG}/<repo>"
```

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @SadriG91 
Could you sign the CLA?

**Comment by comment-bot-dev** [comment]
@SadriG91
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@SadriG91
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**